components/color-picker/doc/index.en-US.md

```diff
@@ -24,38 +24,39 @@ npm install ng-antd-color-picker
-| `[nzValue]`        | Value of color       | `string`｜`NzColor`                  | -         |
-| `[nzSize]`         | Setting the trigger size    | `large`｜`default`｜`small`         | `default` |
-| `[nzDefaultValue]` | Default value of color     | `string`｜`NzColor`                  | `false`   |
-| `[nzAllowClear]`   | Allow clearing color selected  | `boolean`                         | `false`   |
-| `[nzTrigger]`      | ColorPicker trigger mode | `hover`｜`click`                   | `click`   |
-| `[nzShowText]`      | Show color text     | `boolean`                         | `false`   |
-| `[nzOpen]`      | Whether to show popups     | `boolean`                         | `false`  |
-| `[nzDisabled]`     | Disable ColorPicker    | `boolean`                         | `false`   |
-| `[nzTitle]`      | Setting the title of the color picker | `TemplateRef<void>`｜`string`      | -         |
-| `(nzOnChange)`     | Callback when value is changed    | `EventEmitter<{ color: NzColor; format: string }>`            | -         |
-| `(nzOnClear)`      | 	Called when clear      | `EventEmitter<boolean>`           | -         |
-| `(nzOnFormatChange)`      | Callback when `format` is changed      | `EventEmitter<'rgb'｜'hex'｜'hsb'>` | -         |
-| `(nzOnOpenChange)`      | Callback for opening the color panel  | `EventEmitter<boolean>` | -        |
+| Parameter            | Description                           | Type                                               | Default   |
+| -------------------- | ------------------------------------- | -------------------------------------------------- | --------- |
+| `[nzFormat]`         | Format of color                       | `rgb`｜`hex`｜`hsb`                                | `hex`     |
+| `[nzValue]`          | Value of color                        | `string`｜`NzColor`                                | -         |
+| `[nzSize]`           | Setting the trigger size              | `large`｜`default`｜`small`                        | `default` |
+| `[nzDefaultValue]`   | Default value of color                | `string`｜`NzColor`                                | `false`   |
+| `[nzAllowClear]`     | Allow clearing color selected         | `boolean`                                          | `false`   |
+| `[nzTrigger]`        | ColorPicker trigger mode              | `hover`｜`click`                                   | `click`   |
+| `[nzShowText]`       | Show color text                       | `boolean`                                          | `false`   |
+| `[nzOpen]`           | Whether to show popups                | `boolean`                                          | `false`   |
+| `[nzDisabled]`       | Disable ColorPicker                   | `boolean`                                          | `false`   |
+| `[disabledAlpha]`    | Disable Alpha                         | `boolean`                                          | `false`   |
```

Comment:
seems to be `nzDisabledAlpha`

Reply:
You are right, it is nzDisabledAlpha. I have made the necessary changes to the code. Could you please check it again and let me know your feedback?